etc/docker-scripts/bootstrapper.py

```diff
@@ -84,6 +86,7 @@ def parse_arguments(args):
     parser.add_argument('-n', '--notebook', dest="notebook", help='Notebook to execute', required=True)
     parser.add_argument('-o', '--outputs', dest="outputs", help='Files to output to object store', required=False)
     parser.add_argument('-i', '--inputs', dest="inputs", help='Files to pull in from parent node', required=False)
+    parser.add_argument('-r', '--user-volume-path', dest="user-volume-path", help='Directory in Volume to install python libraries into', required=False)
```

Comment:
Just curious why 'r' was selected as the shorter option?  Seems like '-p' or -u' makes more sense.